backend-rust/migrations/0001_initialize.up.sql

```diff
@@ -193,7 +199,7 @@ CREATE TABLE accounts(
         BIGINT
         NOT NULL,
     -- Connect the account with the transaction creating it.
-    FOREIGN KEY (created_block, created_index) REFERENCES transactions(block_height, index)
+    FOREIGN KEY (created_block, created_index) REFERENCES transactions (block_height, block_index)
```

Comment:
Not sure if this should just use the new global index or if it is useful to keep the block/block index way.